source/hic_hal/nxp/lpc11u35/gpio.c

```diff
@@ -94,11 +94,13 @@ void gpio_init(void)
     PIN_CDC_LED_IOCON = PIN_CDC_LED_IOCON_INIT;
     LPC_GPIO->SET[PIN_CDC_LED_PORT] = PIN_CDC_LED;
     LPC_GPIO->DIR[PIN_CDC_LED_PORT] |= PIN_CDC_LED;
+#ifndef GPIO_RESET_IN_NOT_SUPPORTED
```

Comment:
This is common code so this define should be removed.  Instead, the IO_Config_Override.h mechanism should be used.